yarn.lock

```diff
@@ -3228,7 +3228,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"ansi-html@npm:0.0.7, ansi-html@npm:^0.0.7":
+"ansi-html@npm:0.0.7":
```

Comment:
I see that the 0.0.7 version is still used by "webpack-dev-server@npm:3.11.1", which should be fine since it is the dev server.

Reply:
But this means that the dependabot alert will need to be dismissed, as it will still complain about this usage.